From d1acab17ce8393f0fcab91cc0b73af3f242e7728 Mon Sep 17 00:00:00 2001
From: travankor <travankor@tuta.io>
Date: Fri, 18 Sep 2020 19:04:59 -0700
Subject: [PATCH] Use TOP Layer.

The OVERLAY layer is used by lock screens and notifications.
Wlogout can appear over lock screens which may be a security issue.
---
 main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Patch-Source: https://github.com/ArtsyMacaw/wlogout/pull/15

diff --git a/main.c b/main.c
index 4c7eeb0..9973145 100644
--- a/main.c
+++ b/main.c
@@ -282,7 +282,7 @@ static GtkWidget *get_window()
     else {
         #ifdef LAYERSHELL
         gtk_layer_init_for_window (window);
-        gtk_layer_set_layer (window, GTK_LAYER_SHELL_LAYER_OVERLAY);
+        gtk_layer_set_layer (window, GTK_LAYER_SHELL_LAYER_TOP);
         gtk_layer_set_exclusive_zone (window, exclusive_level);
         gtk_layer_set_keyboard_interactivity (window, TRUE);
 
